### ISSUE TITLE
Use a more reliable website to crawl in tutorial

### ISSUE THREAD

**Issue by IdiosApps** [elixir-crawly/crawly#209]
The tutorial uses the homebase site. I think the site layout has changed - for example, there's no body returned for this code in the tutorial:
```
iex(3)> response = Crawly.fetch("https://www.homebase.co.uk/our-range/tools")
%HTTPoison.Response{
  body: "",
```

A more reliable alternative might be some options from
- https://www.scrapethissite.com/pages/
- https://webscraper.io/test-sites

**Comment by oltarasenko** [comment]
Yeah, I have removed the tutorial in favour of the quickstart. Maybe having more sites in the examples folder would be our better target
